SQL/sql.txt

```diff
@@ -0,0 +1,43 @@
+--CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+create table employees(emp_id int primary key,first_name varchar(30),last_name varchar(30),department varchar(30),salary int);
+
+--INSERT: Adds three employee records to the table.
+insert into employees values(1,'a','abin','cse',20000);
```

Comment:
it is always better to mention the column name while inserting data into a table

Reply:
ok